extensions-core/azure-extensions/src/main/java/org/apache/druid/storage/azure/AzureUtils.java

```diff
@@ -136,4 +137,11 @@ public static void deleteObjectsInPath(
       }
     }
   }
+
+  // Returns a <containerName, path> pair given a location using the azureStorage schema.
+  public static Pair<String, String> parseAzureStorageLocation(CloudObjectLocation location)
```

Comment:
The usage of this seems to assume that the location is provided is somehow generated from the new AzureStorageInputSource, is that right? Given this should this maybe be a static method of `AzureStorageInputSource` class, or should we add to the javadoc here the assumption being made?